pinc/showstartexts.inc

```diff
@@ -38,9 +38,8 @@ function showstartexts($etext_limit, $type)
     // Display feed buttons
     echo "
         <a href='$code_url/feeds/backend.php?content=$content'>
-        <img src='$code_url/graphics/xml.gif' width='36' height='14' alt='[XML]'></a>
-        <a href='$code_url/feeds/backend.php?content=$content&amp;type=rss'>
-        <img src='$code_url/graphics/rss.gif' width='36' height='14' alt='[RSS]'></a>
+        <i class='fas fa-rss-square rss-icon'></i>
```

Comment:
Good question. I followed [fontawesome documentation](https://fontawesome.com/v5/icons/rss-square?f=classic&s=solid), without digging into the underlying mechanism. I also saw other examples in the code that already use `<i>` for fontawesome. Just now I also found documentation that says you can use `<span>` instead of `<i>`, and maybe other elements too, but fontawesome recommends using one element type consistently.

Reply:
😁 -- shows you how alert I am, or I'd have seen it in earlier fa-related code. If I'd noticed then, I'd probably have suggested `<span>` just out of persnicketyness (since it doesn't imply a particular treatment of the enclosed character), but I agree. Whatever we use, we should be consistent.

Reply:
Yes, let's stick with `<i>` as that's what we use most other places.